libs/ui/src/components/MediaItem/type/ImageMedia.vue

```diff
@@ -81,9 +83,9 @@ const loading = computed<'eager' | 'lazy'>(() =>
 )
 
 const className = computed(() =>
-  !props.original && !props.isFullscreen
+  `${(!props.original && !props.isFullscreen
     ? 'object-contain absolute inset-0 w-full h-full !rounded-none'
-    : 'block !rounded-none',
+    : 'block !rounded-none')} ${props.innerClass}`,
```

Comment:
isn't it easier to have backticks only on the second statment?

Reply:
updated